app/scripts/run-apis.sh

```diff
@@ -15,10 +15,22 @@ docker run --rm \
     $registry/montagu-cert-tool:$cert_tool_version \
     gen-keypair /workspace
 
+docker volume prune -f
```

Comment:
If the orderly volume hasn't been cleaned up for any reason, the below test generation task will fail

Reply:
We shouldn't prune all volumes. It's fine for test/development tasks to stop and remove containers. But volumes are meant to be the persistent part. If the developer has anything on their machine in a volume that they actually want they could lose it by running this test.

Maybe you could just `docker volume rm` the appropriate volume?